### PR TITLE
Fix static data updates

### PR DESCRIPTION
**Pull request by hannahker**
Minor updates to ensure that the static data update runs from the GitHub action with informative logging and status updates. 

Issues still to resolve: 

- [x] Connection errors when running action ([for example](https://github.com/OCHA-DAP/hdx-signals/actions/runs/9117039573/job/25067015146))
- [x] The action should fail when R errors out

If we're assuming that all files under `inputs/` in the storage container should be updated by [this workflow](https://github.com/OCHA-DAP/hdx-signals/blob/main/.github/workflows/update_assets.yaml):

- [x] `adm0/` 
- [x] `cities/` 
- [x] `country_info.parquet`
- [x] `hrp_countries.parquet` 
- [x] `iso3_map_settings.json`
- [x] `centroids/` -- Updates work locally but not in GH runner
- [x] `indicator_mapping.parquet`
- [x] `idmc_country_links.parquet` 
- [ ] `acled_info.parquet` -- Updates work locally but not in GH runner
- [ ] `un_geodata.geojson` -- Updated manually and used in the code. Can this be updated programmatically?
- [ ] ~`country_taxonomy_backup.parquet` -- Removing~
- [ ] ~`region_bbox.parquet` -- Removing~
- [ ] ~`region_bbox.geojson` -- Removing~

### CODE REVIEW

**Comment by caldwellst** [comment]
> The action should fail when R errors out

This is an issue with using `find` in GitHub Actions. This is why in the original file we were using `globstar`. See this https://apple.stackexchange.com/questions/49042/how-do-i-make-find-fail-if-exec-fails

**Comment by caldwellst** [comment]
> Connection errors when running action ([for example](https://github.com/OCHA-DAP/hdx-signals/actions/runs/9117039573/job/25067015146))

Does this work when running locally?

**Comment by caldwellst** [comment]
`un_geodata.geojson`: See #88 
`idmc_country_links.parquet` and `indicator_mapping.parquet`. I put those together manually, just checking all country links on the IDMC website, and then piecing together `indicator_mapping.parquet` as I was developing. I think we can hardcode an update script that we adjust as necessary. A really nice and easy trick is that you can generate the code to create the data frame in R super easily. Just load it in, save to `df` and then use `dput(df)`. However, it's not that clean / easy to update, whereas `dplyr::tribble()` is a nice way to rowwise create data.

**Comment by hannahker** [comment]
@caldwellst 

Aha nice find! 

> This is an issue with using find in GitHub Actions. This is why in the original file we were using globstar. See this https://apple.stackexchange.com/questions/49042/how-do-i-make-find-fail-if-exec-fails

It does! I'm thinking that this could be from the `{}` in the `DSCI_AZ_ENDPOINT` environment variable. I'm thinking we can just hard-code this in the code itself since this doesn't contain any sensitive data. 

> Does this work when running locally?

File: .github/workflows/update_assets.yaml [hunk 1]
```diff
@@ -16,6 +16,8 @@ jobs:
         runs-on: ubuntu-latest
         env:
           GMAS_TEST_RUN: ${{ inputs.test || 'FALSE' }}
+          DSCI_AZ_SAS_DEV: ${{ secrets.DSCI_AZ_SAS_DEV }}
```
Comment (caldwellst) [inline .github/workflows/update_assets.yaml:19]: Oh, was `DSCI_AZ_ENDPOINT` also not set here?

Comment (hannahker) [reply .github/workflows/update_assets.yaml:19]: Hah, it was also missing 😆 But I was testing and it still seems to cause some issues even when I put it back. So might just leave as is...

**Comment by hannahker** [comment]
@caldwellst this should be ready for final review now. As discussed, we'll come back to the ACLED data issues in future work. For now, I've left the bash command as using `find`, so that this script can run through all other update files, despite the ACLED data failing.

File: src-static/update_indicator_mapping.R [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+)
+# nolint end
+
+fname <- "input/indicator_mapping.parquet"
+cs$update_az_file(
+  df = df,
+  name = fname
+)
+
+log_info(paste0("Successfully downloaded indicator mapping and saved to ", fname))
```
Comment (caldwellst) [inline src-static/update_indicator_mapping.R:32]: Ooh sorry, can you just put this as "created indicator mapping"

File: src-static/update_acled_info.R [hunk 3]
```diff
@@ -4,46 +4,59 @@ box::use(dplyr)
...
-    acled_hdx_country = Country |>
-      stringr$str_remove_all("[^[:alnum:]\\s-\\.]") |>
-      stringr$str_remove_all("\\.$") |>
-      tolower() |>
-      stringr$str_replace_all("\\s|\\.", "-"),
-    acled_hdx_url = as.character(
-      glue$glue(
-        "https://data.humdata.org/dataset/{acled_hdx_country}-acled-conflict-data"
+# TODO: Remove once this passes from GH runner
+if (interactive()) {
```
[the remark this answers is not in the document]
Comment (hannahker) [reply src-static/update_acled_info.R:13]: yeah, still failing :(

[the remark this answers is not in the document]
Comment (hannahker) [reply src-static/update_acled_info.R:13]: Oh sorry, by that I mean that the ACLED access still doesn't work, but the overall action is successful with `interactive()`

**Pull request merged**